docs/source/resources/migration.md

```diff
@@ -19,7 +47,10 @@
 | `DatasetError`              | `DataSetError`              | `kedro.io.core`                |
 | `DatasetAlreadyExistsError` | `DataSetAlreadyExistsError` | `kedro.io.core`                |
 | `DatasetNotFoundError`      | `DataSetNotFoundError`      | `kedro.io.core`                |
+
 * All other dataset classes are removed from the core Kedro repository (`kedro.extras.datasets`). Install and import them from the [`kedro-datasets`](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-datasets) package instead.
 
 ### Logging
 `logging.yml` is now independent of Kedro's run environment and only used if `KEDRO_LOGGING_CONFIG` is set to point to it.
+
+<!-- TO DO - add example code that illustrates where the changes have occured (which files are affected) -->
```

Comment:
The changes made here were pretty much all non user facing. I found it a bit hard to explain so just linked to the full docs if users want to know more about how logging works.